src/webview/webViewEventHandlers.js

```diff
@@ -278,6 +284,17 @@ export const handleMessageListEvent = (props: Props, _: GetText, event: MessageL
       logging.error('WebView error', event.details);
       break;
 
+    case 'time': {
+      const alertText = _.intl.formatMessage(
+        {
+          id: "This time is in your timezone. Original text was '{originalText}'.",
+          defaultMessage: "This time is in your timezone. Original text was '{originalText}'.",
```

Comment:
This feels redundant, as it's just the same string as `id`. We don't use it in the definition of `_` -- we just pass `id` there. (I searched for `formatMessage` to look for examples.)

Can you take this `defaultMessage` line out, and does it all still work as expected?

Based on a quick look at the API docs:
https://formatjs.io/docs/react-intl/api/#formatmessage
https://formatjs.io/docs/react-intl/api/#message-formatting-fallbacks
they agree with our practice at `_`, that it should work.

Reply:
It works.

Reply:
Hmm, I see. I agree, then. Seems like rather a wart in that API.